pkgs/development/ocaml-modules/ocaml-protoc-plugin/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, fetchFromGitHub, buildDunePackage, ocaml-ng, pkg-config, protobuf }:
```

Comment:
As the package belongs to ocamlPackages, you should take ocaml dependencies by argument instead of taking the whole ocaml-ng as argument.

Reply:
Alright. I did not specify any OCaml version in commit 90a0de189c1682ea50a9984c40817742cd73381f so I am not exactly sure how the correct version is handled except from the minimalOcamlVersion attribute though (in particular, I am not sure the package supports Ocaml 5).